api/tests/unit/infrastructure/serializers/jsonapi/shared-profile-for-campaign-serializer_test.js

```diff
@@ -0,0 +1,120 @@
+const _ = require('lodash');
+
+const { expect, domainBuilder } = require('../../../../test-helper');
+const serializer = require('../../../../../lib/infrastructure/serializers/jsonapi/shared-profile-for-campaign-serializer');
+
+describe.only('Unit | Serializer | JSONAPI | shared-profile-for-campaign-serializer', () => {
+
+  describe('#serialize()', () => {
+    const expectedScoreCards = [
```

Comment:
C'est en un seul mot ;) 
```suggestion
    const expectedScorecards = [
```